src/node_url.cc

```diff
@@ -325,7 +325,75 @@ const uint8_t C0_CONTROL_ENCODE_SET[32] = {
     0x01 | 0x02 | 0x04 | 0x08 | 0x10 | 0x20 | 0x40 | 0x80
 };
 
-const uint8_t PATH_ENCODE_SET[32] = {
+static const uint8_t FRAGMENT_ENCODE_SET[32] = {
```

Comment:
Seems like there was some issues in rebasing: the `static` here and below should have been removed when rebasing. This could be taken care of this when landing.